signac/__main__.py

```diff
@@ -1042,13 +1042,20 @@ def jobs():
             if READLINE:
                 if 'PyPy' not in platform.python_implementation():
                     fn_hist = project.fn('.signac_shell_history')
-                    try:
```

Comment:
I'd like to keep the more *Pythonic* try-except approach to address this issue. The code is Python 3-only, so you can take full advantage of the [ `FileNotFoundError`](https://docs.python.org/3/library/exceptions.html#FileNotFoundError) and [`PermissionError`](https://docs.python.org/3/library/exceptions.html#PermissionError) exceptions.

Reply:
@csadorf ya sure, will update it accordingly!

Reply:
Great, please re-request review once you're done. Thx!